src/plots/cartesian/set_convert.js

```diff
@@ -67,24 +82,170 @@ module.exports = function setConvert(ax) {
         else return BADNUM;
     }
 
-    function fromLog(v) {
-        return Math.pow(10, v);
+    /*
+     * wrapped dateTime2ms that:
+     * - accepts ms numbers for backward compatibility
+     * - inserts a dummy arg so calendar is the 3rd arg (see notes below).
```

Comment:
I knew there was a good reason. Thanks for the info!

I still think we could come up with a better pattern here. Having to remember to pass `ax.calendar` in the trace module (or transform?) calc steps could potentially produce bugs in future PRs. 

Let me test out a few things. Anyhow, I'm _ok_ with the current implementation.